SwiftUIDZ/SwiftUIDZ/ContentView.swift

```diff
@@ -7,10 +7,107 @@
 
 import SwiftUI
 
+/// Экран примера алертов
 struct ContentView: View {
+    
+    private enum Constants {
+        static let firstAlertTitleText = "Пример Alert"
+        static let showButtonTitle = "Показать"
+        static let firstAlertTitle = "Первый алерт"
+        static let secondAlertTitleText = "Пример Alert c 2 кнопками и логикой"
+        static let secondAlertSecondTitleText = "Новый текст"
+        static let secondAlertTitle = "Второй алерт"
+        static let secondAlertMessageText = "Поменять текст в лейбле"
+        static let cancelButtonTitle = "Закрыть"
+        static let secondAlertSecondaryButtonTitle = "Поменять"
+        static let actionSheetTitleText = "Пример ActionSheet"
+        static let secondActionSheetTitleText = "Пример ActionSheet c кнопками и логикой"
+        static let secondActionSheetButtonTitle = "Прибавить число"
+        static let newNumberText = "Новое число ="
+    }
+    
+    @State private var presentingFirstAlert = false
+    @State private var presentingSecondAlert = false
+    @State private var textUpdate = false
+    @State private var presentFirstActionSheet = false
+    @State private var presentSecondActionSheet = false
```

Comment:
```suggestion
    @State private var isFirstAlertShown = false
    @State private var isSecondAlertShown = false
    @State private var textUpdate = false
    @State private var isFirstActionSheetShown = false
    @State private var isSecondActionSheetShown = false
```
если флаг говорит, что надо что-то показать начинается с is заканчивается  Shown